.kokoro/presubmit/java10.cfg

```diff
@@ -0,0 +1,7 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
+    key: "TRAMPOLINE_IMAGE"
+    value: "gcr.io/cloud-devrel-kokoro-resources/java11"
```

Comment:
This doesn't look right - java11 for the java10 job?

Reply:
Whoops, you're right, I was testing the java11 image without having to make internal changes. Will fix this.